resources/views/layouts/user.blade.php

```diff
@@ -0,0 +1,92 @@
+<!doctype html>
+<html lang="{{ str_replace('_', '-', app()->getLocale()) }}">
+<head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <!-- CSRF Token -->
+    <meta name="csrf-token" content="{{ csrf_token() }}">
+
+    <title>{{ config('app.name') }} | @yield('title')</title>
+
+    <!-- Fonts -->
+    <link rel="dns-prefetch" href="//fonts.bunny.net">
+    <link href="https://fonts.bunny.net/css?family=Nunito" rel="stylesheet">
```

Comment:
Can download this? Prevent using links in href or src

Reply:
I Commented out this part. But previous Pull Request had this part and approved.

Reply:
Can change this please apply this in all of the href and src.